doc/tutorial/tutorial.md

```diff
@@ -892,12 +892,28 @@ BestPrice(item, best_price) :-
 
 Here `group_min()` is one of several aggregation functions defined in
 `lib/std.rs`.  What if we wanted to return the name of the vendor along with the
-lowest price for each item?  This requires an aggregation function that takes a
+lowest price for each item?  The following naive attempt will not work:
+
+```
+output relation BestVendor(item: string, vendor: string, price: bit<64>)
```

Comment:
I think that a diagram would be more helpful than the text explanation.
We can talk about it in person.

Reply:
Merging for now.  This is better than nothing. We can add diagram later.